src/actions/archiveThought.ts

```diff
@@ -93,7 +93,8 @@ const archiveThought = (state: State, options: { path?: Path }): State => {
   /** Gets the previous sibling context in the context view. */
   const prevContext = () => {
     const thoughtsContextView = thoughtsEditingFromChain(state, path)
-    const contexts = showContexts ? getContextsSortedAndRanked(state, headValue(state, thoughtsContextView)) : []
+    const head = headValue(state, thoughtsContextView)
```

Comment:
Let's just call this `value`. I don't want it to be confused with the `head` function.

Reply:
@snqb